src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ContestedOrderController.java

```diff
@@ -64,6 +64,8 @@
 
         contestedCaseOrderService.printAndMailGeneralOrderToParties(caseDetails, authToken);
 
+        contestedCaseOrderService.printAndMailHearingDocuments(caseDetails, authToken);
```

Comment:
does this need toggled off or is it toggled somewhere else?

also, is there a chance we could send 2 print packs here?

Reply:
As far as I am aware, this does not require a toggle. I have asked Grace to confirm! Why do we want to trigger the print packs from the controller?

Reply:
did you get a response on this?

I meant it there a chance that the above method call sends docs to print and this method call sends docs to print? I would assume we only want to send each person 1 pack